### PR TITLE
Various software lists: Cleaned up more extraneous spaces.

### PR DESCRIPTION
**Pull request by 0kmg**
Note there's a weird Unicode code point after the word "Turbo" in the second changed line in a800.xml. It has been removed but doesn't show up on GitHub's diff. Same issue in this file popped up in #8112.

### CODE REVIEW

**Comment by 0kmg** [comment]
@Tafoid Here are some more if you haven't already found them.

**Comment by Tafoid** [comment]
I am working on larger lists as my commit message said "more to come".  I was hoping you would wait until I was finished so as to not duplicate effort just never found the time to verify/send them all at once.

**Comment by Tafoid** [comment]
I've posted what I'm going to for now. https://github.com/mamedev/mame/commit/f93bf1c5f3a1ad1b6fd97705cdcfd678d068bb23

File: hash/a800.xml [hunk 1]
```diff
@@ -2794,7 +2794,7 @@ Compiled by K1W1
 
 	<software name="jawbrek2">
 		<description>Jawbreaker II</description>
-		<!-- The title screen says “Jawbreaker (c) 1980”, however this actually is Jawbreaker II from 1982, as seen on the box (not in all releases though). -->
+		<!-- The title screen says "Jawbreaker (c) 1980", however this actually is Jawbreaker II from 1982, as seen on the box (not in all releases though). -->
```
Comment (cuavas) [inline hash/a800.xml:2797]: This is unnecessary – no reason to be scared of typographical quotes.

**Pull request merged**